modules/ducktests/tests/ignitetest/services/utils/jmx_utils.py

```diff
@@ -96,7 +96,7 @@ def mbean_attribute(self, mbean, attr):
         return iter(s.strip() for s in self.__run_cmd(cmd))
 
     def __run_cmd(self, cmd):
-        return self.node.account.ssh_capture(cmd, allow_fail=False, callback=str)
+        return self.node.account.ssh_capture(cmd, allow_fail=False, callback=str, combine_stderr=False)
```

Comment:
Why this change?

Reply:
This is just a "easy fix" to run this function from the JDK.
This function takes the first line from the console output.
If we use JDK> = 9 then the first line will be a warning from JDK in stderr
Since we don't really need stderr, we can ignore it and workaround this situation